src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs

```diff
@@ -393,7 +393,7 @@ public bool TryFetch(LocalUserRegistryOptionPersister persister, OptionKey2 opti
         {"dotnet_show_navigation_bar", new RoamingProfileStorage("TextEditor.%LANGUAGE%.Dropdown Bar", vbKey: "TextEditor.Basic.Dropdown Bar")},
 #pragma warning restore
         {"dotnet_include_navigation_hints_in_quick_info", new RoamingProfileStorage("TextEditor.Specific.IncludeNavigationHintsInQuickInfo")},
-        {"dotnet_show_remarks_in_quick_info", new RoamingProfileStorage("TextEditor.%LANGUAGE%.Specific.ShowRemarks")},
+        {"dotnet_include_remarks_in_quick_info", new RoamingProfileStorage("TextEditor.%LANGUAGE%.Specific.ShowRemarks")},
```

Comment:
Looking at all the options here I think it would be better to rename `dotnet_include_navigation_hints_in_quick_info` to `dotnet_show_navigation_hints_in_quick_info` and keep `dotnet_show_remarks_in_quick_info`.

That way we reduce the number of verbs we use (i.e. not use `include`) as we use `show` in a few other options as well.